### PR TITLE
fix feature file quality flaws

### PR DESCRIPTION
**Pull request by Bertk**
- language profile file: gherkin-rules.xml
- analyze with https://github.com/racodond/sonar-gherkin-plugin

---
This change is [Reviewable](https://reviewable.io/reviews/sonaropencommunity/sonar-cxx/1238)

### CODE REVIEW

**Comment by guwirth** [comment]
@Bertk what exactly is this PR doing:
* improve formating Behave future files?
* What is this gherkin-rules.xml? How does this fit to the caption?

**Comment by Bertk** [comment]
@guwirth This PR fixes issue identified by the [SonarQube Gherkin plugin](https://github.com/racodond/sonar-gherkin-plugin) (quality profile gherkin-rules.xml):
- enable SQ analysis with maven (pom.xml)
- removed parser issues of **_Eclipse plugin for Cucumber**_
- correct scenario syntax to use only one _When_ or _Then_ statement

_**list of fixed issues**_
fixed-issues

_Maybe we should create a separat folder for recommended quality profiles (Java, XML, Gherkin, ...)_

File: integration-tests/gherkin-rules.xml [hunk 1]
```diff
@@ -0,0 +1,312 @@
+<?xml version='1.0' encoding='UTF-8'?>
```
Comment (guwirth) [inline integration-tests/gherkin-rules.xml]: @Bertk would move this file to
* https://github.com/SonarOpenCommunity/sonar-cxx/tree/master/integration-tests/features.
* add a comment at the beginning that this file is to use [SonarQube Gherkin plugin](https://github.com/racodond/sonar-gherkin-plugin)

Comment (Bertk) [reply integration-tests/gherkin-rules.xml]: :ok:

**Comment by Bertk** [comment]
@guwirth This PR has to be merged before PR #1237.

**Pull request merged**